client-participation/js/strings/sk.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2012-present, The Authors. This program is free software: you can redistribute it and/or  modify it under the terms of the GNU Affero General Public License, version 3, as published by the Free Software Foundation. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+var s = {};
+
+s.agree = "Súhlasím";
+s.disagree = "Nesúhlasím";
+s.pass = "Zdržiavam sa / Nie som si istý/á";
```

Comment:
Might you be able to read the first post here, and confirm that your translation feels like the most appropriate? https://github.com/pol-is/polis/discussions/774

Thanks so much!

Reply:
Yes, I think it covers most of the things in that list :)

Reply:
Ok, cool! This is the longest string for "pass" we have so far, so slightly worried it might break user interface, especially since agree/disagree are also amongst the longest.

But if it can't be said with less characters, then we'll have to just work with it :)